pkg/requeue/requeue.go

```diff
@@ -21,6 +21,11 @@ const (
 	DefaultRequeueAfterDuration time.Duration = 30 * time.Second
 )
 
+// GetDurationInSeconds converts an integer into a time duration in seconds
+func GetDurationInSeconds(duration int) time.Duration {
```

Comment:
nit: I am not a big fan of this method. From the name it implies like minutes or hours being converted into seconds.
Why not just use inline like time.Duration(x) * time.Second instead of creating this public method.

Reply:
done.